### ISSUE TITLE
[SF] ColorPalettePopover - [TAB] and [ENTER] behaviour and screen reader accessibility issues for the "More Colors" dialog

### ISSUE THREAD

**Issue by Vir-Mittal** [SAP/ui5-webcomponents#5015]
### **Bug Description**
1. When keyboard navigation is used to access the ColorPalettePopover, the behaviour seen is confusing/not expected. Pressing [TAB] on the "Default Color" button and any of the "Color Items" selects the color and closes the dialog. Pressing [ENTER] selects the color but does not close the dialog
2.  The two sliders in the "More Colors" dialog are difficult to distinguish when using a screen reader such as NVDA as the description for both is "Slider handle slider". So, it is difficult to know their purpose. When focusing on/editing the hex value or RGBA values, the name/description for the value being edited is also not announced.

### **Expected Behavior**
1. Pressing [TAB] should shift the focus to the next focusable element without closing the dialog. Pressing [ENTER] on a color palette item within the dialog should select that color and close the dialog.
2. The "More Colors" option dialog should be more screen reader friendly by providing more accessible, relevant names for the sliders and input boxes

### **Steps to Reproduce**
1. Go to https://codesandbox.io/s/0im72p?file=/src/App.js
3. Use keyboard navigation within the ColorPalettePopover dialog
4. Press [TAB] - the popover closes and a color is picked
5. Open the dialog, focus on a color palette item using arrow keys and then press [ENTER]
6. The color is selected but the dialog remains open
7. Use the "More Colors" option dialog using NVDA

### **Isolated Example**
https://codesandbox.io/s/0im72p?file=/src/App.js

### **Context**
- UI5 Web Components version: {1.2.4}
- OS/Platform: {MacOS and Windows}
- Browser: {Chrome}

**Comment by nnaydenow** [comment]
Hello @Vir-Mittal,

I'm assigning @unazko as accessibility expert of ui5-color-pallete to analyse the issue.

Regards,
Nayden

**Comment by rchrdchn** [comment]
Hey guys - following up here. Any updates on when a potential fix will be available?

**Comment by unazko** [comment]
Hello @rchrdchn,

The second issue here is the same to what is reported in #5023.
Regarding the first one I can confirm that the current behavior is compliant with the keyboard specification.
When navigating in the ui5-colorpicker the following keys could be used:
- HOME, END to navigate between "Default Colors" and "More Colors" sections
- Arrow keys to navigate in the grid of swatches

Expect updates soon regarding the second issue.

Best Regards,
Boyan Rakilovski

**Comment by unazko** [comment]
Hello @rchrdchn,

The related fix https://github.com/SAP/ui5-webcomponents/pull/5217 is now merged.
For the first issue as TAB keyboard key is expected to close the dialog by specification
you can use the following shortcut for fast navigation:
- HOME, END to navigate between "Default Colors" and "More Colors" sections,

Best Regards,
Boyan Rakilovski
